base/expvar.go

```diff
@@ -136,17 +138,11 @@ const (
 	StatKeyQueryProcessingTime     = "query_processing_time"
 
 	// StatsReplication
-	StatKeyNumDocsTransferred       = "num_docs_transferred"
-	StatKeyNumDocsTransferredPerSec = "num_docs_transferred_per_sec"
-	StatKeyBandwidth                = "bandwidth"
-	StatKeyDataReplicatedSize       = "data_replicated_size"
-	StatKeyNumAttachmentsTransfered = "num_attachments_transferred"
-	StatKeyNumTempFailures          = "num_temp_failures"
-	StatKeyNumPermFailures          = "num_perm_failures"
-	StatKeyPendingBacklog           = "pending_backlog"
-	StatKeyBatchSize                = "batchsize"
-	StatKeyDocTransferLatency       = "doc_transfer_latency"
-	StatKeyDocsCheckedSent          = "docs_checked_sent"
+	StatKeyNumDocsPushed              = "num_docs_pushed"
+	StatKeyNumDocsFailedToPush        = "num_docs_failed_to_push"
+	StatKeyNumAttachmentsTransferred  = "num_attachments_transferred"
+	StatKeyAttachmentBytesTransferred = "num_attachment_bytes_transferred"
+	StatKeyDocsCheckedSent            = "docs_checked_sent"
```

Comment:
A few of these constants look like they might have naming collisions (if not exact, then close enough to cause confusion/accidental use of the wrong stat) with non-sg-replicate stats.  Should we prefix the constant names with Sgr in those cases?